src/product/domain/values/price.vo.ts

```
@@ -0,0 +1,76 @@
+  static of(
+    priceWithoutTax: Money,
+    taxRate: number = PRICE_CONSTANTS.DEFAULT_TAX_RATE,
+  ): Price {
+    return new Price(priceWithoutTax, taxRate);
+  }
+
+  static withTax(
+    priceWithTax: Money,
+    taxRate: number = PRICE_CONSTANTS.DEFAULT_TAX_RATE,
+  ): Price {
+    const priceWithoutTax = this.calculatePriceWithoutTax(
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)